avaframe/in3Utils/generateTopoCfg.ini

```diff
@@ -0,0 +1,80 @@
+### Config File - This file contains the main settings for the simulation run
+## Set your parameters
+# This file is part of Avaframe.
+
+
+# Domain parameters----------------------
+[GENERAL]
+dx = 5.
+x_end = 5000
+y_end = 1500
+
+# dx                  # DEM spatial resolution [m]
+# x_end               # total horizontal extent of the domain [m]
+# y_end               # total horizontal extent of the domain [m]
+#------------------------------------------------------
+
+# Topography parameters-----------------------
+[TOPO]
+f_lens = 2250
+mean_alpha = 24
+C = 1000
+r_bowl = 500
+r_helix = 1250
+z0 = 2200
+z_elev = 0
+r_circ = 200
+DEM_type = HS
```

Comment:
See comment in generateTopo.py regarding type:
List the possible options and their meaning here and in the docu...

Reply:
listed all topography types in the .ini file